### PR TITLE
update oui kit : components

### PR DESCRIPTION
**Pull request by virgotopia**

### CODE REVIEW

File: client/components/telecom/telephony/alias/records/telecom-telephony-alias-records.html [hunk 1]
```diff
@@ -330,7 +330,9 @@
                     <tbody data-ng-if="$ctrl.records.isLoading">
                         <tr>
                             <td colspan="7">
-                                <oui-loader data-size="m"></oui-loader>
+                                <div class="text-center">
+                                    <oui-spinner></oui-spinner>
```
Comment (antleblanc) [inline client/components/telecom/telephony/alias/records/telecom-telephony-alias-records.html]: +`data-size="s"`

File: client/components/telecom/telephony/group/number/feature/ovhPabx/menu/list/telephony-group-number-feature-ovh-pabx-menu-list.html [hunk 2]
```diff
@@ -31,7 +31,9 @@
     <tbody data-ng-if="$ctrl.loading.init">
         <tr>
             <td colspan="3">
-                <oui-loader data-size="m"></oui-loader>
+                <div class="text-center">
+                    <oui-spinner></oui-spinner>
```
Comment (antleblanc) [inline client/components/telecom/telephony/group/number/feature/ovhPabx/menu/list/telephony-group-number-feature-ovh-pabx-menu-list.html]: +`data-size="s"`.

File: client/components/telecom/telephony/group/number/feature/ovhPabx/sound/list/telephony-group-number-feature-ovh-pabx-sound-list.html [hunk 3]
```diff
@@ -26,7 +26,9 @@
     <tbody data-ng-if="$ctrl.loading.init">
         <tr>
             <td colspan="3">
-                <oui-loader data-size="m"></oui-loader>
+                <div class="text-center">
+                    <oui-spinner></oui-spinner>
```
Comment (antleblanc) [inline client/components/telecom/telephony/group/number/feature/ovhPabx/sound/list/telephony-group-number-feature-ovh-pabx-sound-list.html]: +`data-size="s"`.

File: client/components/telecom/telephony/alias/records/telecom-telephony-alias-records.html (hunk 1 above)
Comment (Jisay) [reply client/components/telecom/telephony/alias/records/telecom-telephony-alias-records.html]: Mmmmh... Indead spinner size into array should be size s. But this rule will probably be applied later with `oui-datagrid` implementation ?

File: client/components/telecom/alias/hunting/sounds/telecom-telephony-alias-hunting-sounds.html [hunk 4]
```diff
@@ -27,10 +27,10 @@ <h3 data-translate="telephony_alias_hunting_sounds_manager"></h3>
                 class="btn btn-default mt-2 float-right no-transition"
                 data-ng-disabled="$ctrl.isUploading"
                 data-ng-if="$ctrl.toUpload">
-            <oui-loader class="mr-2"
+            <oui-spinner class="mr-2"
                         data-ng-if="$ctrl.isUploading"
```
Comment (Jisay) [inline client/components/telecom/alias/hunting/sounds/telecom-telephony-alias-hunting-sounds.html]: Take care to attributes alignment

File: client/components/telecom/telephony/abbreviatedNumbers/telecom-telephony-abbreviated-numbers.html [hunk 5]
```diff
@@ -114,9 +114,9 @@
                               data-ng-bind="abbreviated.name"></span>
                     </td>
                     <td>
-                        <oui-loader data-ng-if="abbreviated.removing"
+                        <oui-spinner data-ng-if="abbreviated.removing"
                                     data-size="s">
```
Comment (Jisay) [inline client/components/telecom/telephony/abbreviatedNumbers/telecom-telephony-abbreviated-numbers.html]: Missing 1 space ;-)

File: client/components/telecom/telephony/alias/liveCalls/eavesdrop/telecom-telephony-alias-configuration-liveCalls-eavesdrop.html [hunk 6]
```diff
@@ -54,10 +54,10 @@
                 <button type="submit"
                         class="btn btn-primary float-right mt-2"
                         data-ng-disabled="eavesdropForm.$invalid">
-                    <oui-loader class="mr-2"
+                    <oui-spinner class="mr-2"
                                 data-ng-if="$ctrl.isSubmitting"
```
Comment (Jisay) [inline client/components/telecom/telephony/alias/liveCalls/eavesdrop/telecom-telephony-alias-configuration-liveCalls-eavesdrop.html]: missing spaces

File: client/components/telecom/telephony/alias/liveCalls/hangup/telecom-telephony-alias-configuration-liveCalls-hangup.html [hunk 7]
```diff
@@ -28,10 +28,10 @@
                 <button type="submit"
                         class="btn btn-primary float-right mt-2"
                         data-ng-disabled="hangupForm.$invalid">
-                    <oui-loader class="mr-2"
+                    <oui-spinner class="mr-2"
                                 data-ng-if="$ctrl.isSubmitting"
```
Comment (Jisay) [inline client/components/telecom/telephony/alias/liveCalls/hangup/telecom-telephony-alias-configuration-liveCalls-hangup.html]: missing spaces

File: client/components/telecom/telephony/alias/liveCalls/intercept/telecom-telephony-alias-configuration-liveCalls-intercept.html [hunk 8]
```diff
@@ -40,10 +40,10 @@
                 <button type="submit"
                         class="btn btn-primary float-right mt-2"
                         data-ng-disabled="interceptForm.$invalid">
-                    <oui-loader class="mr-2"
+                    <oui-spinner class="mr-2"
                                 data-ng-if="$ctrl.isSubmitting"
```
Comment (Jisay) [inline client/components/telecom/telephony/alias/liveCalls/intercept/telecom-telephony-alias-configuration-liveCalls-intercept.html]: missing spaces

File: client/components/telecom/telephony/alias/liveCalls/transfert/telecom-telephony-alias-configuration-liveCalls-transfert.html [hunk 9]
```diff
@@ -40,10 +40,10 @@
                 <button type="submit"
                         class="btn btn-primary float-right mt-2"
                         data-ng-disabled="transfertForm.$invalid">
-                    <oui-loader class="mr-2"
+                    <oui-spinner class="mr-2"
                                 data-ng-if="$ctrl.isSubmitting"
```
Comment (Jisay) [inline client/components/telecom/telephony/alias/liveCalls/transfert/telecom-telephony-alias-configuration-liveCalls-transfert.html]: missing spaces

File: client/components/telecom/telephony/alias/members/telecom-telephony-alias-members-add.html [hunk 10]
```diff
@@ -106,10 +106,10 @@
         <button type="submit"
                 class="btn btn-primary"
                 data-ng-disabled="memberAddForm.$invalid || $ctrl.loaders.adding">
-            <oui-loader class="mr-2"
+            <oui-spinner class="mr-2"
                         data-ng-if="$ctrl.loaders.adding"
```
Comment (Jisay) [inline client/components/telecom/telephony/alias/members/telecom-telephony-alias-members-add.html]: here too

File: client/components/telecom/telephony/alias/members/telecom-telephony-alias-members.html [hunk 11]
```diff
@@ -59,9 +59,9 @@
                  data-ng-class="{'pl-5': !$ctrl.memberInEdition && !$ctrl.memberToDelete}">
                 <strong class="member-description my-3  align-middle"
                         data-ng-class="{'px-4': !$ctrl.memberInEdition && !$ctrl.memberToDelete}">
-                    <oui-loader data-ng-if="member.description === undefined"
+                    <oui-spinner data-ng-if="member.description === undefined"
                                 data-size="s">
```
Comment (Jisay) [inline client/components/telecom/telephony/alias/members/telecom-telephony-alias-members.html]: 1 space more :-)

File: client/components/telecom/telephony/alias/members/telecom-telephony-alias-members.html [hunk 12]
```diff
@@ -149,10 +149,10 @@
                         <button type="submit"
                                 class="btn btn-primary"
                                 data-ng-disabled="$ctrl.loaders.editing">
-                            <oui-loader class="mr-2"
+                            <oui-spinner class="mr-2"
                                         data-ng-if="$ctrl.loaders.editing"
```
Comment (Jisay) [inline client/components/telecom/telephony/alias/members/telecom-telephony-alias-members.html]: missing spaces

File: client/components/telecom/telephony/alias/members/telecom-telephony-alias-members.html [hunk 13]
```diff
@@ -171,10 +171,10 @@
                             class="btn btn-danger"
                             data-ng-click="$ctrl.deleteMember(member)"
                             data-ng-disabled="$ctrl.loaders.deleting">
-                        <oui-loader class="mr-2"
+                        <oui-spinner class="mr-2"
                                     data-ng-if="$ctrl.loaders.deleting"
```
Comment (Jisay) [inline client/components/telecom/telephony/alias/members/telecom-telephony-alias-members.html]: missing spaces

File: client/components/telecom/telephony/alias/records/telecom-telephony-alias-records.html [hunk 14]
```diff
@@ -126,10 +126,10 @@
                         class="btn btn-primary no-transition"
                         data-ng-if="$ctrl.hasChanges()"
                         data-ng-disabled="ovhPabxRecordsOptionsForm.$invalid || $ctrl.queues.isUpdating">
-                        <oui-loader class="mr-2"
+                        <oui-spinner class="mr-2"
                                     data-ng-if="$ctrl.isUpdating"
```
Comment (Jisay) [inline client/components/telecom/telephony/alias/records/telecom-telephony-alias-records.html]: spaces here too

File: client/components/telecom/telephony/alias/records/telecom-telephony-alias-records.html [hunk 15]
```diff
@@ -158,10 +158,10 @@
                 class="btn btn-default mb-3"
                 data-ng-disabled="$ctrl.getSelection().length === 0 || $ctrl.records.isDeleting"
                 data-ng-click="$ctrl.deleteSelectedRecords()">
-                <oui-loader class="mr-2"
+                <oui-spinner class="mr-2"
                             data-ng-if="$ctrl.records.isDeleting"
```
Comment (Jisay) [inline client/components/telecom/telephony/alias/records/telecom-telephony-alias-records.html]: spaces

File: client/components/telecom/telephony/associateDevice/telecom-telephony-associate-device.html [hunk 16]
```diff
@@ -65,10 +65,10 @@
         <button type="submit"
                 class="btn btn-primary float-right"
                 data-ng-disabled="$ctrl.isAttaching || !$ctrl.selectedMac || !$ctrl.ipAddress">
-            <oui-loader class="mr-2"
+            <oui-spinner class="mr-2"
                         data-ng-if="$ctrl.isAttaching"
```
Comment (Jisay) [inline client/components/telecom/telephony/associateDevice/telecom-telephony-associate-device.html]: spaces

File: client/components/telecom/telephony/callsFiltering/addHelper/telecom-telephony-callsFilteringAddHelper.html [hunk 17]
```diff
@@ -60,10 +60,10 @@ <h3 data-translate="telephony_calls_filtering_helper_title"></h3>
         <button type="submit"
                 class="btn btn-default"
                 data-ng-disabled="filteringHelperAddForm.$invalid || FilteringHelperCtrl.isLoading">
-            <oui-loader class="mr-2"
+            <oui-spinner class="mr-2"
                         data-ng-if="FilteringHelperCtrl.isLoading"
```
Comment (Jisay) [inline client/components/telecom/telephony/callsFiltering/addHelper/telecom-telephony-callsFilteringAddHelper.html]: spaces

File: client/components/telecom/telephony/callsFiltering/telecom-telephony-callsFilteringAdd.html [hunk 18]
```diff
@@ -97,10 +97,10 @@
             <button type="submit"
                     class="btn btn-primary"
                     data-ng-disabled="$ctrl.screenListToAdd.isAdding || filteringAddForm.$invalid || $ctrl.isScreenListsAlreadyExisting()">
-                <oui-loader class="mr-2"
+                <oui-spinner class="mr-2"
                             data-ng-if="$ctrl.screenListToAdd.isAdding"
```
Comment (Jisay) [inline client/components/telecom/telephony/callsFiltering/telecom-telephony-callsFilteringAdd.html]: spaces

File: client/components/telecom/telephony/callsFiltering/telecom-telephony-callsFilteringTable.html [hunk 19]
```diff
@@ -7,10 +7,10 @@
                     type="button"
                     data-ng-disabled="$ctrl.getSelection().length === 0 || $ctrl.screenLists.isDeleting"
                     data-ng-click="$ctrl.removeSelectedScreenLists()">
-                <oui-loader class="mr-2"
+                <oui-spinner class="mr-2"
                             data-ng-if="$ctrl.screenLists.isDeleting"
```
Comment (Jisay) [inline client/components/telecom/telephony/callsFiltering/telecom-telephony-callsFilteringTable.html]: here too

File: client/components/telecom/telephony/callsFiltering/telecom-telephony-callsFilteringTable.html [hunk 20]
```diff
@@ -185,18 +185,20 @@
                             <i class="text-success ovh-font ovh-font-success"
                                data-ng-if="screen.status === 'active'">
                             </i>
-                            <oui-loader class="ml-1"
+                            <oui-spinner class="ml-1"
                                         data-ng-if="screen.status !== 'active'"
```
Comment (Jisay) [inline client/components/telecom/telephony/callsFiltering/telecom-telephony-callsFilteringTable.html]: one more space

File: client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html [hunk 21]
```diff
@@ -249,9 +249,9 @@ <h2 data-translate="telephony_number_feature_conference_popover_title"></h2>
                                   data-translate="none">
                             </span>
                             <div data-ng-if="$ctrl.loading.webAccess">
-                                <oui-loader class="mr-2"
+                                <oui-spinner class="mr-2"
                                             data-size="s">
```
Comment (Jisay) [inline client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html]: space

File: client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html [hunk 22]
```diff
@@ -272,9 +272,9 @@ <h2 data-translate="telephony_number_feature_conference_popover_title"></h2>
...
```
Comment (Jisay) [inline client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html]: indent

File: client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html [hunk 23]
```diff
@@ -339,9 +339,9 @@ <h2 data-translate="telephony_number_feature_conference_popover_title"></h2>
                         <div class="d-table">
                             <div class="d-table-cell align-middle px-3">
                                 <i class="ovh-font ovh-font-upload fs18" data-ng-if="!$ctrl.loading.announceUpload"></i>
-                                <oui-loader data-ng-if="$ctrl.loading.announceUpload"
+                                <oui-spinner data-ng-if="$ctrl.loading.announceUpload"
                                             data-size="s">
```
Comment (Jisay) [inline client/components/telecom/telephony/group/number/feature/conference/telephony-group-number-feature-conference-configuration.html]: indent plz

File: client/components/telecom/telephony/group/number/feature/ovhPabx/tts/create/telephony-group-number-feature-ovh-pabx-tts-create.html [hunk 24]
```diff
@@ -55,10 +55,10 @@
             <button type="submit"
                     class="btn btn-primary float-right ml-sm-2"
                     data-ng-disabled="$ctrl.loading.creating || !$ctrl.ttsCreateForm.$valid">
-                <oui-loader class="mr-2"
+                <oui-spinner class="mr-2"
                             data-ng-show="$ctrl.loading.creating"
```
Comment (Jisay) [inline client/components/telecom/telephony/group/number/feature/ovhPabx/tts/create/telephony-group-number-feature-ovh-pabx-tts-create.html]: here too

File: client/components/telecom/telephony/group/number/telephony-group-number.html [hunk 25]
```diff
@@ -112,10 +112,10 @@ <h2 class="voip-plan__title" data-translate="telephony_number_incoming_calls"></
             <button type="submit"
                     class="btn btn-primary"
                     data-ng-disabled="!$ctrl.number.feature.hasChange() || $ctrl.loading.save || $ctrl.loading.feature">
-                <oui-loader class="mr-2"
+                <oui-spinner class="mr-2"
                             data-ng-if="$ctrl.loading.save"
```
Comment (Jisay) [inline client/components/telecom/telephony/group/number/telephony-group-number.html]: one space to attributes

File: client/components/telecom/telephony/scheduler/telephony-scheduler-actions.html [hunk 26]
```diff
@@ -115,10 +115,10 @@
                         <button type="button"
                                 class="btn btn-primary float-right ml-2"
                                 data-ng-click="$ctrl.manageDeleteAll()">
-                            <oui-loader class="mr-2"
+                            <oui-spinner class="mr-2"
                                         data-ng-if="$ctrl.loading.deleteAll"
```
Comment (Jisay) [inline client/components/telecom/telephony/scheduler/telephony-scheduler-actions.html]: here too

File: client/components/telecom/telephony/scheduler/telephony-scheduler.html [hunk 27]
```diff
@@ -271,10 +271,10 @@ <h3 data-ng-bind="$ctrl.getCalendarTitle()">
             <button type="submit"
                     class="btn btn-primary"
                     data-ng-disabled="$ctrl.loading.save">
-                <oui-loader class="mr-2"
+                <oui-spinner class="mr-2"
                             data-ng-if="$ctrl.loading.save"
```
Comment (Jisay) [inline client/components/telecom/telephony/scheduler/telephony-scheduler.html]: here too

**Pull request merged**